eblocker-icapserver/src/main/java/org/eblocker/server/common/blacklist/Cache.java

```diff
@@ -53,16 +48,20 @@ class Cache {
     private final String cacheIndexFile;
     @Nonnull
     private final ObjectMapper objectMapper;
+    @Nonnull
+    private final CacheIndex index;
 
-    private CacheIndex index;
+    private final CacheIndexRepository cacheIndexRepository;
```

Comment:
This class is responsible for reading and writing the index. It seems to me to be more readable than before.